client/src/helpers/getDownloadOptionDetails.js

```diff
@@ -28,8 +29,15 @@ const formatFileItemByKey = (key, computedFile) => {
 // takes the config and checks against the resource
 // to present what will be inside of the downloadable file
 export const getDownloadOptionDetails = (computedFile) => {
-  const { modality, project, sample } = computedFile
-  const type = project ? 'Project' : 'Sample'
+  const {
+    metadata_only: metadataOnly,
+    modality,
+    project,
+    sample
+  } = computedFile
+  // eslint-disable-next-line no-nested-ternary
+  const type = metadataOnly ? 'Sample Metadata' : project ? 'Project' : 'Sample'
+  const isSampleMetadata = type === 'Sample Metadata'
```

Comment:
Can we remove `isSampleMetadata` or rename `metadataOnly` since they are (and should always be) the same value?

Reply:
This is to say do *one* of the following:
- remove `isSampleMetadata` and use `metadataOnly` here
*or*
- rename `metadataOnly` to `isSampleMetadata`

Reply:
I've removed `isSampleMetadata` and use  `metadataOnly`, see the commit at ba188e8760d703c7c361fcad3b6a0750ae685411.